.pre-commit-config.yaml

```
@@ -57,5 +57,12 @@ repos:
         args: [--prose-wrap=always, --print-width=88]
         exclude: ^.github/ISSUE_TEMPLATE/bug_report.md$
 
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: "v0.982"
+    hooks:
+      - id: mypy
```

For these:

```
setup.py:3: error: Skipping analyzing "setuptools": module is installed, but
missing library stubs or py.typed marker  [import]
    from setuptools import setup

src/prettytable/colortable.py:6: error: Cannot find implementation or library
stub for module named "colorama"  [import]
        from colorama import init
```

This is because we're importing the setuptools and Colorama modules but they don't have type hints (their tracking issues: https://github.com/pypa/setuptools/issues/2345, https://github.com/tartley/colorama/issues/206).

However, just like PrettyTable didn't type hints before https://github.com/jazzband/prettytable/pull/205, there are stubs published by Typeshed and available at https://pypi.org/project/types-colorama/ and https://pypi.org/project/types-setuptools/

So we can make use of those by telling pre-commit to install it like this:

```suggestion
      - id: mypy
        additional_dependencies: [types-colorama, types-setuptools]
```

(If running mypy outside of pre-commit, `pip install types-colorama types-setuptools`)